weakmw.tex

```diff
@@ -201,23 +250,32 @@ \section{Proof of the Weak Mordell-Weil Theorem}
 	injective uses \emph{formal groups}, whose development is outside the
 	scope of this course.\footnote{For an introduction to
 	formal groups of elliptic curves see \cite[Ch. IV]{silverman:aec}.}
-	Next, as above, $\sigma(Q)-Q \in E(K)[n]$ for
+	Next fix any $Q$ as in the statement of the theorem.
+	Above we saw $\sigma(Q)-Q \in E[n]$ for
 	all $\sigma \in \Gal(\Kbar/K)$.  Let $I_\p\subset \Gal(L/K)$ be the
-	inertia group at $\p$.  Then by definition of interia group, $I_\p$
+	inertia group at $\p$. By definition of interia group, $I_\p$
 	acts trivially on $\tilde{E}(\O_K/\p)$.  Thus for each $\sigma \in
 	I_\p$ we have
 	$$
 		\pi(\sigma(Q) - Q) = \sigma(\pi(Q)) - \pi(Q) = \pi(Q) - \pi(Q) = 0.
 	$$
 	Since $\pi$ is injective, it follows that $\sigma(Q) = Q$ for $\sigma\in I_\p$,
-	i.e., that $Q$ is fixed under all $I_\p$.  This means that the subfield
-	of~$L$ generated by the coordinates of $Q$ is unramified at $\p$.
-	Repeating this argument with all choices of~$Q$ implies that $L$
-	is unramified at~$\p$.
+	i.e., that $Q$ is fixed under all $I_\p$.
+	Repeating for all $Q$ this shows $I_\p = 1$ so $L$ is unramified
+	at $\p$.
+
+	%TODO weave this into the proof following Silverman or
+	%TODO reword \pi ?
+	Note that we technically only defined $\pi$ on $E(K)[n]$ and
+	$\sigma(Q)-Q$ may not lie in $E(K)$. However, given a finite
+	extension $K'/K$ and prime $\p'$ lying over $\p$, $E$ the
```

Comment:
VERY WRONG -- critical that we be right.  In particular, worry about p' being ramified...    (State our thing correctly.)